tests/shutdown/grub_set_bootargs.pm

```diff
@@ -16,16 +16,32 @@ use testapi;
 
 sub run() {
     select_console('root-console');
-    script_run "source /etc/default/grub";
-    script_run 'new_cmdline=`echo $GRUB_CMDLINE_LINUX_DEFAULT | sed \'s/\(^\| \)quiet\($\| \)/ /\'`';
-    if (get_var("ENABLE_FIPS")) {
-        script_run 'new_cmdline="$new_cmdline fips=1"';
-        if (get_var("ENCRYPT") && !get_var("FULL_LVM_ENCRYPT")) {
-            script_run 'new_cmdline="$new_cmdline boot=$(df /boot | tail -1 | cut -d" " -f1)"';
+    my @cmds;
+    push @cmds, "source /etc/default/grub";
+    push @cmds, 'new_cmdline=`echo $GRUB_CMDLINE_LINUX_DEFAULT | sed \'s/\(^\| \)quiet\($\| \)/ /\'`';
+    push @cmds, 'sed -i "s#GRUB_CMDLINE_LINUX_DEFAULT=\"[^\"]*\"#GRUB_CMDLINE_LINUX_DEFAULT=\"$new_cmdline\"#" /etc/default/grub';
+    push @cmds, 'grub2-mkconfig -o /boot/grub2/grub.cfg';
+
+    # Slow type for 12-SP2 aarch64 image creatation test to try to avoid filling up the key event queue
+    if (check_var('ARCH', 'aarch64') && check_var('VERSION', '12-SP2')) {
+        for my $cmd (@cmds) {
+            type_string $cmd . "\n", wait_screen_change => 1;
```

Comment:
In the past, we have created many sle12 images including GA, sp1,2,3 across on different archs, and I only see the issue happens with 12sp2 aarch64 image creation, even 12sp3 aarch64 image could be published successfully.

Furthermore, I think type_string with wait_screen_change parameter is not sufficient to run command, since it doesn't check status of command. Maybe a better way to hack script_run with adding a slow type option?